scripts/stackOperation.py

```diff
@@ -0,0 +1,70 @@
+class Stack:
+    def __init__(self):
+        self.stack = []
+
+    def is_empty(self):
+        return not self.stack
+
+    def push(self, item):
+        self.stack.append(item)
+
+    def pop(self):
+        if self.is_empty():
+            return 'Underflow'
+        else:
+            item = self.stack.pop()
+            return item
+
+    def peek(self):
+        if self.is_empty():
+            return 'Underflow'
+        else:
+            top = len(self.stack) - 1
+            return self.stack[top]
+
+    def __repr__(self):
+        if self.is_empty():
+            return 'Stack is empty'
+        else:
+            top = len(self.stack) - 1
+            stack_str = f'{self.stack[top]} <-top\n'
+            for i in range(top - 1, -1, -1):
+                stack_str += str(self.stack[i]) + '\n'
+            return stack_str
+
+def main():
+    stack = Stack()
+
+    while True:
+        print('\n\nStack Operations:')
+        print('1. Push')
+        print('2. Pop')
+        print('3. Peek')
+        print('4. Display Stack')
+        print('5. Exit')
+        ch = int(input('Enter your choice:'))
```

Comment:
What happens if you enter something that's not a number? We could wrap this line in a `try except` here to warn the user instead of crashing the whole program. Something like

```py
try:
    ch = int(input('Enter your choice:'))
except ValueError:
    print("Incorrect entry")
    continue
```

Reply:
do I change it again or leave it as it is?

Reply:
I'd recommend the change